test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -937,7 +937,7 @@ func unrollWorkloadTemplate(b *testing.B, wt []op, w *workload) []op {
 	return unrolled
 }
 
-func runWorkload(ctx context.Context, b *testing.B, tc *testCase, w *workload) []DataItem {
+func runWorkload(ctx context.Context, b *testing.B, tc *testCase, w *workload, cleanup bool) []DataItem {
```

Comment:
Should we make it an option? Considering we'll setup an etcd instance for each case, I think we can remove them directly. I created a separate PR here, but includes in this one also sgtm. You're the boss. https://github.com/kubernetes/kubernetes/pull/116843

Reply:
As I am not going to get this PR merged soon, I continued with that unit test in the same branch and just pushed an update. Perhaps review commit-by-commit and then before we merge, I take out everything after the commit which adds the DRA test cases and move the rest into a separate PR?